content/030_eksctl/console.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Console Credentials"
+date: 2018-08-07T13:36:57-07:00
+weight: 40
+---
+
+This step is optional, as nearly all of the workshop content is CLI-driven. But, if you'd like full access to your workshop cluster in the EKS console this step is recommended.
+
+The EKS console allows you to see not only the configuration aspects of your cluster, but also to view Kubernetes cluster objects such as Deployments, Pods, and Nodes. For this type of access, the console IAM User or Role needs to be granted permission within the cluster.
+
+By default, the credentials used to create the cluster are automatically granted these permissions. Following along in the workshop with temporary IAM credentials in Cloud9 means that you'll need to manually add your unrelated AWS Console credentials.
+
+#### Import your EKS Console credentials to your new cluster:
+
+IAM Users and Roles are bound to the cluster via a ConfigMap named `aws-auth`. We can use `eksctl` to do this with one command. You'll need the IAM credential ARN from your console session, e.g. arn:aws:iam::510431938379:role/Admin.
+
+```bash
+ARN=<Your console ARN here>
```

Comment:
Maybe We can add specific section here for customers doing the workshop at an AWS Events.
In this case we already know the Role ARN use in the console so they can directly execute : 
```bash
ACCOUNT_ID=$(aws sts get-caller-identity | jq -r '.Account')
eksctl create iamidentitymapping \
  --cluster eksworkshop-eksctl \
  --arn arn:aws:iam::${ACCOUNT_ID}:role/TeamRole \
  --username TeamRole \
  --group system:masters
```

Reply:
Good idea. Added in a passage to determine the arn of either IAM user or role. Good call out, thanks!

Reply:
@jlbutler seems nice, thanks